### PR TITLE
Feature/sales overview database

### PR DESCRIPTION
**Pull request by armen-h**
Implement linking between activity screen and database (using callbacks provided by interfaces). The items that are actually stored on the database appear on a scrollable view displaying title and price.
Title is clickable and directs to the details of the item for sale.

### CODE REVIEW

File: app/src/main/java/ch/epfl/polybazaar/SalesOverview.java [hunk 1]
```diff
@@ -2,29 +2,186 @@
...
+
+import static ch.epfl.polybazaar.litelisting.LiteListingDatabase.fetchLiteListing;
+import static ch.epfl.polybazaar.litelisting.LiteListingDatabase.fetchLiteListingList;
 
 public class SalesOverview extends AppCompatActivity {
 
+    // TODO: add tests for error cases (ex empty list on database)
+
+    private List<String> IDList;
+    private List<LiteListing> liteListingList;
```
Comment (m-caboche) [inline app/src/main/java/ch/epfl/polybazaar/SalesOverview.java]: Might want to consider using a separate class for this in the future. Could be like a class that is filled as data is loaded and can be read once the data is ready. But that's just something to keep in mind for future expansion; right now, this works just fine.
